javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,123 @@
+function getReply(command) {
+  let result;
+  if (command.includes("hello my name is")) {
+    const userName = saveName(command);
+    result = `nice to meet you ${userName}`;
+  } else if (command.includes(str2)) {
+    if (!str1) {
+      result = `you have not tell your name yet`;
+    }
+    const yourName = saveName(str1);
```

Comment:
This is not very dynamic, as it will just take your name from the string variable, and not from what you speak after pressing the button. Better way would be to pass "command" and pick up the last word from it.

Reply:
Yes, I have changed the code to make it accepts dynamic input.